dspace-api/pom.xml

```diff
@@ -673,7 +673,12 @@
             <artifactId>hibernate-jpa-2.1-api</artifactId>
             <version>1.0.0.Final</version>
         </dependency>
-
+        <dependency>
```

Comment:
This PR is far behind, currently DSpace 7 requires at least Java 11 and it's not backward compatible. This PR should be upgraded.
I'm getting this compiling error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project dspace-services: Compilation failure
[ERROR] DSpace2655_git/dspace-services/src/main/java/org/dspace/servicemanager/config/DSpaceConfigurationPlaceholderConfigurer.java:[27,8] cannot access java.lang.Object
[ERROR] bad class file: /modules/java.base/java/lang/Object.class
```